doc/chapters/integration-guidelines/index.rst

```diff
@@ -20,7 +19,13 @@ Service manifests
 For details about format and content of service manifests, see :ref:`Service manifests <service-manifests>`
 and :ref:`Gateways <gateways>`.
 
-Service manifests should be installed in ``/etc/softwarecontainer/caps.d/<service-name>``.
+Service manifests should be installed in ``/etc/softwarecontainer/caps.d/<service-name>`` and are
+expected to be of type JSON, including the "json" file extension.
+
+Service Manifests are read and parsed at startup of the Agent. If two Service Manifests contain
+capabilities with the same name the gateway configurations will be combined (without merging
+or removing duplicates), so that when the capability's gateway configurations are set all
+configurations from the manifests are included.
```

Comment:
You could refer to some part of the docs that explain what the policy is for conflicting/overlapping GW configs. I guess the Gateway chapter should mention that.